nomad/structs/structs.go

```diff
@@ -6038,10 +6033,6 @@ func (t *Template) Validate() error {
 		}
 	}
 
-	if t.VaultGrace.Nanoseconds() < 0 {
```

Comment:
Move to a warning: https://github.com/hashicorp/nomad/blob/v0.10.3/nomad/structs/structs.go#L5700-L5709

Something like:

```go
if t.Template != nil && t.Template.VaultGrace > 0 {
	mErr.Errors = append(mErr.Errors, fmt.Errorf("VaultGrace has been deprecated as of Nomad 0.11 and ignored since Vault 0.5. Please remove VaultGrace / vault_grace from template stanza."))
```